js/private/js_run_binary.bzl

```diff
@@ -275,6 +279,24 @@ See https://github.com/aspect-build/rules_js/tree/main/docs#using-binaries-publi
             name = to_label(name),
         ))
 
+    # Configure run from execroot
+    if run_from_execroot:
```

Comment:
there's no complex magic in the implementation, that's nice :)

Reply:
layering is still working out nicely

Reply:
💎